PoC-differ/differ.go

```diff
@@ -177,7 +177,7 @@ func processReportsByCluster(ruleContent map[string]types.RuleContent, impacts t
 			continue
 		}
 
-		notificationMsg := generateNotificationMessage(notificationConfig.ClusterDetailsURI, string(cluster.AccountNumber), string(cluster.ClusterName))
+		notificationMsg := generateNotificationMessage(notificationConfig.ClusterDetailsURI, fmt.Sprint(cluster.AccountNumber), fmt.Sprint(cluster.ClusterName))
```

Comment:
it is more time consuming operation - is it really needed there?

Reply:
Not needed at all. I let sed do the job, and this happened ^^

Reply:
:)